bot/resources/tags/async-await.md

```diff
@@ -2,18 +2,18 @@
 
 Python provides the ability to run multiple tasks and coroutines simultaneously with the use of the `asyncio` library, which is included in the Python standard library.
 
-This works by running these coroutines in an event loop, where the context of which coroutine is being run is switches periodically to allow all of them to run, giving the appearance of running at the same time. This is different to using threads or processes in that all code is run in the main process and thread, although it is possible to run coroutines in threads.
+This works by running these coroutines in an event loop, where the context of the running coroutine switches periodically to allow all the coroutines to run, thus giving the appearance of running at the same time. This is different to using threads or processes in that all code runs in the main process and thread, although it is possible to run coroutines in threads.
```

Comment:
```suggestion
This works by running these coroutines in an event loop, where the context of the running coroutine switches periodically to allow all of the coroutines to run, thus giving the appearance of running at the same time. This is different to using threads or processes in that all code runs in the main process and thread, although it is possible to run coroutines in threads.
```

Reply:
PyCharm doesn't like the added `of` here, likely since it's somewhat grammatically redundant, which is why I removed it.

![image](https://user-images.githubusercontent.com/47674925/134577074-74e9146a-ec86-4919-a4c5-86edd78b9617.png)

Reply:
Sweet, I'm no English major so I'm good with it. If I say it out load I want to use of, but that could just be me. 👍